src/main/java/me/zeroeightsix/fiber/annotation/convention/SettingNamingConvention.java

```diff
@@ -1,5 +1,13 @@
 package me.zeroeightsix.fiber.annotation.convention;
 
+/**
+ * A setting naming convention decides how a setting is named based upon the variable it was created from.
+ * <br> Java fields are usually {@code lowerCamelCase}, while developers might want their configuration to use {@code snake_case} or {@code lowercase}.
+ * Classes implementing this interface are expected to make the conversion from {@code lowerCamelCase} to another format.
```

Comment:
Expected to, but also required to?

Reply:
I'll change *expected to* to *required to*.